README.md

```diff
@@ -44,6 +44,17 @@ Spyder may also be used as a PyQt4/PySide extension library (module
 `spyderlib`). For example, the Python interactive shell widget used in
 Spyder may be embedded in your own PyQt4/PySide application.
 
+## Running from source
+
+The fastest way to run Spyder is to get the source code, install PyQt
+or PySide, and run:
+
+    python bootstrap.py
+
+You may want to do this for fixing bugs in Spyder, adding new
+features, learning how Spyder works or just getting a taste of it.
+Make sure you use a stable branch though, because `master` may not
+always be clean.
```

Comment:
master is not always clean, but I can't remember a time where spyder didn't work at all. The precision here seems too much.